src/CrawlAbstractLvl/private/snake.h

```diff
@@ -0,0 +1,30 @@
+//#
+//# Copyright (C) 2021-2021 QuasarApp.
+//# Distributed under the GPLv3 software license, see the accompanying
+//# Everyone is permitted to copy and distribute verbatim copies
+//# of this license document, but changing it is not allowed.
+//#
+
+#ifndef SNAKE_H
```

Comment:
This class will be conflict with Snake class of the core module. so please rename your snake class.